inductiva/fluids/fluid_block/fluid_block.py

```diff
@@ -62,15 +62,20 @@ def __init__(self,
         """Initializes the fluid block scenario.
 
         Args:
-            density: The density of the fluid in kg/m^3.
-            kinematic_viscosity: The kinematic viscosity of the fluid, in m^2/s.
+            density: The density of the fluid in kg/m^3. Valid
+                range is [400, 2000] Kg/m3.
```

Comment:
Will those limits work? The error I got from DualSPhysics was:

```
*** Exception (JSphCpuSingle::LoadConfigCommands) at DualSPHysics/source/JSph.cpp:865
Text: The reference density value 1.000000 is outside the defined limits [700.000000,1300.000000].
```

Reply:
Yes, these will work for now - we look into some of the physical aspects of fluids and SpliSHPlasH to make further sense of the values.

However, I want to highlight that DualSPH performance is hindered by density values further way from those of water, which is 1000 Kg/m^3. However, since this is more like a toy scenario I think we can keep this values and it will still work!